controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/role/PathGroup.java

```diff
@@ -53,7 +55,8 @@
                Optional.of("/api"),
                "/application/v4/tenant/{tenant}/application/",
                "/application/v4/tenant/{tenant}/cost",
-               "/application/v4/tenant/{tenant}/cost/{date}"),
+               "/application/v4/tenant/{tenant}/cost/{date}",
+               "/routing/v1/status/tenant/{tenant}"),
```

Comment:
Seems to me everything under `/status/` is `GET`, and could lie here, with `.../{tenant}/{*}`.

Reply:
Thanks!